File: Luminar Python prmgs/Oops/Multiple_Inheritance.py
class Parent:
    def m1(self):
        print("Inside Parent")

class Child():
    def m2(self):
        print("Inside Child")

class SubChild(Child,Parent):
    def m3(self):
        print("Inside Subchild")

p=Parent()
p.m1()
print("\n")

c=Child()
c.m2()
print("\n")
s=SubChild()
s.m3()
s.m2()
s.m1()

File: Luminar Python prmgs/Javascript Progrms/OOP/multipleInheritance.js

//multiple inheritance will not suppport in Javascript...



class Parent{
    phone(){
        console.log("parent have 0301000");

    }
}

class ChildOne extends Parent{
    n1(){
        console.log("Inside CHild One");
     }

}
class Child extends Parent,ChildOne{
}

let obj=new Child();

obj.phone();